### ISSUE TITLE
[React Wrappers] react and jsdoc mixup private function

### ISSUE THREAD

**Issue by pascalvos** [break-stuff/cem-tools#62]
Hey Burton i have found an other small bug

if i have a method like this 
```
#myPrivateMethod() {

}
```
this will work it will be private and have no problems it makes this place 
```
  useImperativeHandle(forwardedRef, () => ({}));
```

```
/*
* This is my private method
*/
#myPrivateMethod() {

}
```
this somehow ends up in the wrapper 
```
  /** Methods - uses `useImperativeHandle` hook to pass ref to component */
  useImperativeHandle(forwardedRef, () => ({
    #myPrivateMethod: () => ref.current.#myPrivateMethod(),
  }));
 ```

I can fix this by doing 

```
/*
* @private
* this is my private method
*/
#myPrivateMethod() {

}
```

but it would be nice to skip these private methods that have comments from the wrapper by default

**Comment by pascalvos** [comment]
might be because 
```
   {
              "kind": "method",
              "name": "#myPrivateMethod"
  },
```
it doesn't get marked as private in the `custom-element.json`

**Comment by break-stuff** [comment]
Oh, interesting. It looks like the Analyzer doesn't mark those as private. I should be able to add a patch for that in my repo until they fix it on their end. Thank you for bringing it to my attention!

**Comment by break-stuff** [comment]
A fix for this has been deployed in release 1.2.0 (#67). Let me know if you run into any issues.

**Comment by break-stuff** [comment]
As a point of reference, [a bug](https://github.com/open-wc/custom-elements-manifest/issues/232) has been filed with the CEM Analyzer to mark these methods as `private`.
